src/main/java/net/minecraftforge/registries/GameData.java

```diff
@@ -899,4 +902,9 @@ public static ResourceLocation checkPrefix(String name, boolean warnOverrides)
         }
         return new ResourceLocation(prefix, name);
     }
+
+    public static boolean isRegistrationCompleted()
+    {
+        return registrationCompleted;
+    }
```

Comment:
Currently talking this over with other triage members since we do have a `ModLoadingPhase` that would make more sense to generally expose rather than a one-off boolean.

Reply:
I would argue that exposing `IModLoadingState` and making it possible to check the relation between two of those would be more useful in the long run as each `ModLoadingPhase` encompasses a fairly large section of the loading process and would only be useful here by pure luck due to "fill registries" being the last task of `ModLoadingPhase.GATHER`.